editor.planx.uk/src/@planx/components/PlanningConstraints/Presentational.tsx

```diff
@@ -80,7 +81,10 @@ export function Presentational(props: PresentationalProps) {
               />
             </SimpleExpand>
           )}
-          <Disclaimer text={disclaimer} />
+          <Disclaimer
+            text={disclaimer}
+            defaultText={DEFAULT_PLANNING_CONDITIONS_DISCLAIMER}
+          />
```

Comment:
nit: what's your reasoning for a separate prop rather than simply passing `text={disclaimer || DEFAULT_PLANNING_CONDITIONS_DISCLAIMER}` to the existing prop?

Would that be a more accurate way to ensure the required "text" prop on the `Disclaimer` component is actually always populated?

Reply:
Thanks @jessicamcinchak :) I did consider this but the since the disclaimer component is shared across a few places (for example, it appears in the Feedback component), it would be weird to have the `defaultText` related to planning conditions.
But you're right, since this is an optional prop, the problem that August encountered could still happen again if someone forgets to account for the case when `text` is `undefined`. 🤔 Perhaps it could be a non-optional prop, or I could come up with a more generic fallback to populate by default in the shared component?

Reply:
Not suggesting that the planning conditions default text is used as the _default_ for the shared component, just that the shared component _only_ needs a single required `text` prop as originally implemented and that this could have been a one-line fix here (in `Presentational.tsx`) I think by passing `text={disclaimer || DEFAULT_PLANNING_CONDITIONS_DISCLAIMER}` (aka _not_ impacting other uses of shared `Disclaimer`)!

Sorry first comment didn't articulate that so clearly

Reply:
Ah sorry I misunderstood, that makes perfect sense!